modules/ztp-deploying-a-site.adoc

```diff
@@ -182,6 +182,6 @@ The `SiteConfig` CR creates the following CRs on the hub cluster:
 * `ClusterDeployment`
 * `InfraEnv`
 * `NMStateConfig` - One per node
-* `ExtraManifestsConfigMap` - Extra manifests. The additional manifests include workload partitioning, chronyd, mountpoint hiding, sctp enablement, and more.
+* `ExtraManifestsConfigMap` - Extra manifests. The additional manifests include workload partitioning, mountpoint hiding, sctp enablement, and more. The extra manifests are automatically merged into a single manifest per each `MachineConfigPool` role, which is named as `predefined-extra-manifests-<role>`.
```

Comment:
Consider s/additional manifests/extra manifests for clarity. 
Presently, the sentence starts with `Extra manifests` then refers to them as `additional manifests` and then back again to `extra manifests` in the next line. While I understand that it's all the same, yet being consistent might help.